apps/frontend/src/app/3_organisms/ConversionsHistoryFrame/ConvertHistory.utils.tsx

```diff
@@ -0,0 +1,24 @@
+import { t } from 'i18next';
+
+import { translations } from '../../../locales/i18n';
+import { ConvertHistoryType } from './ConvertHistory.types';
+
+export const convertHistoryOptions = [
```

Comment:
This doesn't belong to `utils` but to `constants`. Delete this file, rename the const to `CONVERT_HISTORY_OPTIONS` and move it to `ConvertHistory.constants`.